CMakeLists.txt

```diff
@@ -20,10 +20,6 @@ include(GNUInstallDirs)
 option(COMPILE_WITHYARP_DEF "Compile diagnosticdaemon for YARP connection" OFF)
 option(DIAGNOSTICDAEMON_USE_EXTERNAL_PUGIXML "Set to on if you want to use an external pugixml" OFF)
 
-set(Boost_NO_BOOST_CMAKE ON)
```

Comment:
I remember that I had some problems before adding these three lines. But I've never studied in deep. So perhaps the problem was different. Ok for me.

Reply:
@triccyx can you give it a try?

For me these changes are ok but I never used the daemon

Reply:
Yes, if there are any problems feel free to let me know, I suspect we could find a less invasive solution.

Reply:
Ok I will try today.